src/Jackett.Common/Indexers/Anidex.cs

```diff
@@ -14,9 +14,11 @@
 using Jackett.Common.Models.IndexerConfig;
 using Jackett.Common.Services.Interfaces;
 using Jackett.Common.Utils;
+using Jackett.Common.Utils.Clients;
```

Comment:
my vs2019 tells me that `using Jackett.Common.Utils.Clients;` and `using Polly;` are redundant